### PR TITLE
Add SSTMapInfo

### PR DESCRIPTION
**Pull request by lboyd93**
- Adds SSTMapInfo.tsx
- Updates all `Map` files to `SSTMap` to be uniform with the rest of the app.
- Add images to img folder

### CODE REVIEW

**Comment by lboyd93** [comment]
@benelan - this PR is the one that has the images that still need styling. Want me to merge first or wanna make edits on this branch?

**Comment by benelan** [comment]
> @benelan - this PR is the one that has the images that still need styling. Want me to merge first or wanna make edits on this branch?

I'll do it on this branch and then merge when its all good

File: src/components/maps/SSTMapInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    </div>
+    <InfoItem>
+      <h3>
+        <b>Effects of Global Sea Temperatures Rising</b>
+      </h3>
+      <img
+        alt="Effects of global sea temperatures rising."
+        src="../../../public/img/EmissionsTrends.png"
+        style={{}}
+      />
```
Comment (lboyd93) [inline src/components/maps/SSTMapInfo.tsx]: @benelan - one image

**Comment by lboyd93** [comment]
@benelan - just updated the images and need some of your styling expertise again. I want these two to be next to each other but the image seems to be too big and needs to be scaled down a little. Tried adding `objectFit` property to the style but that didn't work..

image

**Pull request merged**